js/script.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-unused-vars */
+const booksArray = [];
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+// const  form
+
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function myFunction() {
+  return false;
+}
+
+function renderBook() {
+  const bookContainer = document.getElementById('booksContainer');
+  bookContainer.innerHTML = '';
+  for (let i = 0; i < booksArray.length; i += 1) {
+    const card = document.createElement('div');
+
+    const cardBody = document.createElement('div');
+
+    const title = document.createElement('h3');
+    title.innerText = booksArray[i].title;
+
+    const author = document.createElement('p');
+    author.innerText = booksArray[i].author;
+
+    cardBody.appendChild(title);
+    cardBody.appendChild(author);
+
+    card.appendChild(cardBody);
+
+    const deleteCard = document.createElement('button');
+    deleteCard.innerText = 'Delete';
+
+    cardBody.appendChild(deleteCard);
+
+    bookContainer.appendChild(card);
+  }
+}
+
+const addBook = () => {
+  const newBook = new Book(title.value, author.value);
+  booksArray.push(newBook);
+  renderBook();
+};
```

Comment:
- [ ] You have a function to add a book, but you need also a function to delete the book, please add it.

- [ ] All the data must be preserved in the local storage, please add a function for this(hint: the preserveData function must be called after a book is added it and when a book is deleted)

Reply:
- [ ]  The delete function doesn't work as correctly. You should also consider books with the same title (or I can have multiple copies of the same book). Let's say I have The Double from Fyodor Dostoevsky and The Double from José Saramago, this method will remove both of them when I try to remove one of them.